app/src/components/nftCard.tsx

```diff
@@ -97,11 +97,12 @@ export const NftCard = (props: NftCardProps): React.ReactElement => {
                   </Box>
                 )}
                 <Stack.Item growthFactor={1} shrinkFactor={1}>
-                  {collectionUrl ? (
-                    <MarkdownText textVariant='small' source={`Part of [${collection.name}](${collectionUrl})`} />
-                  ) : (
-                    <Text variant='small'>{`Part of ${collection.name}`}</Text>
-                  )}
+                  { collectionUrl != null && (
```

Comment:
this looks the same as it was on the left just with more code. you need to be checking the collection.name and if it doesnt exist dont render either of these